### ISSUE TITLE
Bookmark functionality

### ISSUE THREAD

**Issue by ishaanv** [bkhpanigha/hh-suttas#25]
Would it be useful to be able to save and group snippets from different suttas?

**Comment by bkhpanigha** [comment]
Yes, that would be a nice feature. If it's not too difficult to implement it would also be ideal if the data can be backed up, i.e. imported/exported, as JSON or similar.

**Comment by ishaanv** [comment]
Yes can be done. On top of importing and exporting one could also potentially merge (add someone's groupings on top of yours). Will try to implement a basic version.

**Comment by ishaanv** [comment]
This ended up being a little more complicated than initially expected. Still hoping to get something out at some point
